Curs04/src/views/index.js

```diff
@@ -1,8 +1,96 @@
 
 window.onload = function(){
-    const result = [];
-    result.push(LoginPage({}))
     const app = document.querySelector('#app');
+    // call LoginPage function and display LoginPage using innerHTML method
     app.innerHTML = LoginPage();
+
+    // get the form by his ID
+   const loginFormSection = document.getElementById("form-login")
+
+    const myObj = {}
+
+   const goToNewEvaluationPage = function(event) {
+        event.preventDefault();
+        // call NewEvaluationPage() which will render the New-evaluation page
+        app.innerHTML = NewEvaluationPage()
+
+        // get evaluationButton by his ID
+        const evaluationButton = document.getElementById("evaluation-page")
+
+        // add "click" event listener
+        evaluationButton.addEventListener("click", goToEvaluationsPage)
+
+        const getSelectorAll = document.getElementsByTagName("select")
+        getSelectorAll.forEach(function(el) {
+           const nameSelector = el.name
+           const valueSelector = el.value  
+
+           myObj[nameSelector] = valueSelector
+
+           console.log(myObj)
+        })
+    }
+
+    // go to evaluation page function
+    const goToEvaluationsPage = function() {
+        // call EvaluationPage function and display EvaluationPage using innerHTML method
+        app.innerHTML = EvaluationsPage();
+        const newEvaluationButton = document.getElementById("new-evaluation-page")
+
+        // add click event listener on newEvaluationButton
+        newEvaluationButton.addEventListener("click", goToNewEvaluationPage)
+    }
+
+    // Create submitButton function
+    const submitButton = function(event) {
+        event.preventDefault();
+        
+        // get button by his ID #submit-button
+        const submitFormButton = document.getElementById("submit-button")
+    }
+
+    // addEventListener on submit
+    submitFormButton.addEventListener("submit", submitButton)
+
+
+    // submitLogin function
+    const submitLogin = function(event) {
+        event.preventDefault();
+
+        // get username input value by his ID
+        const usernameValue = document.getElementById("username-field").value
+
+        // get password input value by his ID
+        const passwordValue = document.getElementById("password-field").value
+
+        // set alert message if the inputs not meet the conditions
+        const displayAlert = function(message){
+            const verify = document.getElementById('alert') || false;
+            if(!verify){
```

Comment:
What happens if the element already exists and you need to show a different error?

Reply:
You can set a new message in this case, in the if condition.

Reply:
if the element exists the instructions in the if will not be executed, right?

Reply:
yes

Reply:
so if you want to add a new message while a message is showing already, what will happen?

Reply:
If I add a new message won't work :)

Reply:
yep, it won't :)